test/e2e/capi_test.go

```diff
@@ -0,0 +1,26 @@
+// +build e2e
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_test
+
+import (
+	"github.com/onsi/ginkgo"
+)
+
+var _ = ginkgo.Describe("functional tests", func() {
```

Comment:
Do we need this file?

Reply:
I mainly added this to facilitate testing of the utilities, but after CAPD and CABPK are in tree I think we'd want to have E2E testing.

Not directly related, but after this is merged I want to refactor the integration tests to use these utilities as well, so I think I'll move them up a directory to avoid confusion for why the integration tests are importing a package from the e2e dir.